### DISCUSSION TITLE
Welcome to Lambda Powertools  Discussions!

### DISCUSSION THREAD

**Discussion by heitorlessa** [aws-powertools/powertools-lambda-python#327] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. This should also help exposing what ideas are accepted, our short-term plans moving forward.

When possible, we'd love to know how you're using this project and where, because we chose not to do any tracking as that would've impacted your app performance.

That said, we hope that you:

* Ask any questions you have on how to best use this project
* Share ideas on new features, enhancements, improvements we can do to docs
* Engage with other community members
* Welcome others and are open-minded. Remember that this is a community we build together 💪

To get started, comment below with an introduction of yourself, where you work, and how you're using Lambda Powertools today!

**Comment by michaelbrewer** [comment]
first
